db/multi-table.sql

```diff
@@ -62,7 +62,7 @@ CREATE TABLE dragonfruit (
     uptime_ms BIGINT,
     batt_mv INTEGER,
     panel_mv INTEGER,
-    apogee_sp215 INTEGER,
+    apogee_sp215 FLOAT,
```

Comment:
I'm not sure what the reasoning was but I was just following the naming convention on the firmware of each box. I'll change them to match.